### ISSUE TITLE
When a user profile email is missing, the Calendar event popover loads forever

### ISSUE THREAD

**Issue by thom4parisot** [SergeyMosin/Appointments#368]
A way to reproduce a bug:

- a person A creates an Appointment schedule has no email in their email field, and their profile is enabled
<img width="1189" alt="image" src="https://user-images.githubusercontent.com/138627/195113938-f7b64a3c-832b-4f26-a020-ead1771ee8d3.png">
- a person B books an appointment with person A
- when person A looks at their appointment in Calendar app, on click, a popover is displayed, a spinner is visible but nothing else happens
- in the deveveloper console, a JavaScript error is displayed (sorry I don't have the exact content at the moment)

It is related to a JavaScript/Vue function which removes the `mailto:` prefix in the popover.

**Comment by SergeyMosin** [comment]
@thom4parisot Thanks for the report, can you please provide version of the calendar app.

**Comment by thom4parisot** [comment]
I think I remember seeing it was Nextcloud Calendar v3.3.2 with Nextcloud v24.0.4.

**Comment by SergeyMosin** [comment]
Yes. I confirm this bug. However, there is nothing I can do on my side. This is a bug in https://github.com/nextcloud/calendar , because I can create an event without appointments app and it will trigger this behavior, additionally I can view these events just fine in a different calendar client.
If an event has 'ORGANIZER` prop and NC user does not have email address this bug can be replicate.

@thom4parisot if you have time please open an issue with https://github.com/nextcloud/calendar team
